leptonai/cli/deployment.py

```diff
@@ -1329,9 +1329,17 @@ def update(
             tokens=tokens,
         ),
         auto_scaler=AutoScaler(
-            scale_down=ScaleDown(no_traffic_timeout=no_traffic_timeout),
-            target_gpu_utilization_percentage=target_gpu_utilization,
-            target_throughput=AutoscalerTargetThroughput(qpm=threshold),
+            scale_down=(
+                ScaleDown(no_traffic_timeout=no_traffic_timeout)
+                if no_traffic_timeout
+                else None
+            ),
+            target_gpu_utilization_percentage=(
+                target_gpu_utilization if autoscale_gpu_util else None
+            ),
+            target_throughput=(
+                AutoscalerTargetThroughput(qpm=threshold) if threshold else None
+            ),
```

Comment:
are we sending `auto_scaler={}` if nothing is set? it is legal but we should avoid that because auto_scaler might be null at the backend, then sending {} would modify the backend to {} and unnecessarily restart replicas